serde/core.py

```diff
@@ -250,7 +258,7 @@ def conv(f: Field, case: Optional[str] = None) -> str:
             raise SerdeError(
                 (f"Unkown case type: {f.case}." f"Pass the name of case supported by 'stringcase' package.")
```

Comment:
I think the auto formatting was a bit confused here, just
```
raise SerdeError(
    f"Unkown case type: {f.case}. Pass the name of case supported by 'stringcase' package."
)
```
should be fine.

Reply:
And on line [256](https://github.com/yukinarit/pyserde/pull/102/files#diff-217837e513e75f90cbaf8744df290da74e5bfc091c4deefd4b8d28705920f37fR256) just `casef = getattr(stringcase, case, None)` should be fine too. The if statement above already covers the or case.

Reply:
OK, fixed!